app/components/moderation-list-row/template.hbs

```diff
@@ -7,16 +7,17 @@
             <div title={{submission.title}} class="submission-info">
                 <div class="submission-title">{{submission.title}}</div>
                 <div class="submission-body">
-                    {{#if (eq submission.reviewsState 'pending')}}
-                        {{statusTimeDate}}
+                    <div>
+                        {{submittedOnLabel}}
                         {{#each firstContributors as |contributor| ~}}
                             <li>{{contributor.users.fullName}}</li>
```

Comment:
There was a [semi-relevant discussion](https://github.com/CenterForOpenScience/ember-osf-web/pull/60#discussion_r163326966) yesterday about shortened contributor lists on the OSF dashboard. It might be worth considering (at some point) updating reviews to match.

Essentially, say "X more" instead of "+ X", and use "and" when appropriate:
> by Sherif
by Sherif and Abram
by Sherif, Abram, and Lauren
by Sherif, Abram, Lauren, and 5 more

Not a blocker for this PR, though. @laurenbarker, thoughts?

Reply:
That approach would be fine bit is a little out of scope for this PR. @sheriefvt if you'd like to make the suggested changes please put it in another PR (same ticket number is fine though)